DQM/L1TMonitorClient/src/L1TOccupancyClient.cc

```diff
@@ -459,7 +457,8 @@ double L1TOccupancyClient::xySymmetry(const ParameterSet& ps,
       cout << "statsup= " << statsup << ", statslow= " << statslow << endl;
     }
 
-    enoughStats = TMath::MinElement(nActualStrips, maxAvgs.get()) > TMath::Max(statsup, statslow);
+    if (nActualStrips > 0)
+      enoughStats = TMath::MinElement(nActualStrips, maxAvgs.get()) > TMath::Max(statsup, statslow);
     if (verbose_) {
       cout << "stats: " << TMath::MinElement(nActualStrips, maxAvgs.get())
```

Comment:
@cms-sw/dqm-l2 , any idea what should be printed in case of `nActualStrips=0` ?

Reply:
My guess would be something alog these lines
```
if (nActualStrips == 0) {
    cout << "No valid strips found, insufficient statistics." << endl;
  } else {
  ```
  but perhaps the @cms-sw/l1-l2 can confirm